Assets/LeapMotionModules/ElementRenderer/Scripts/Editor/LeapGuiElementEditor.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEditor;
+using Leap.Unity;
+using Leap.Unity.Query;
+
+[CanEditMultipleObjects]
+[CustomEditor(typeof(LeapGuiElementData), editorForChildClasses: true, isFallback = true)]
+public class DefaultFeatureDataEditor : CustomEditorBase {
+  protected override void OnEnable() {
+    base.OnEnable();
+    dontShowScriptField();
```

Comment:
Ah that is right, I was just following the style that already existed in CustomEditorBase, but that file was created a long time ago and needs to be updated to the new style guide !